File: src/com/lstudio/algorithms/ls/Node.java
package com.lstudio.algorithms.ls;

class Node {
    int NodeId;
    int demand; //Node Demand if Customer
    boolean IsRouted;
    boolean isStartDepot;
    boolean isEndDepot;

    Node(int id, int demand, boolean isStartDepot, boolean isEndDepot) //Cunstructor for Customers
    {
        this.NodeId = id;
        this.demand = demand;
        this.IsRouted = false;
        this.isEndDepot = isEndDepot;
        this.isStartDepot = isStartDepot;
    }
}

File: src/main/java/com/lstudio/algorithms/ls/model/Node.kt
package com.lstudio.algorithms.ls.model

internal class Node(
    val nodeId: Int,
    val demand: Int,
    val isStartDepot: Boolean,
    val isEndDepot: Boolean,
    var isRouted: Boolean = false
) {
    override fun toString(): String {
        return "$nodeId"
    }
}